third_party/move/move-compiler-v2/transactional-tests/tests/evaluation_order/struct_arguments.exp

```diff
@@ -0,0 +1,69 @@
+comparison between v1 and v2 failed:
```

Comment:
I assume you'll fix these failed tests at a later stage, right?

Reply:
Yes! https://github.com/aptos-labs/aptos-core/issues/9343